app/SearchService.js

```diff
@@ -0,0 +1,68 @@
+var numberOfSongsOfList0 = collectionPlaylistObjects[0].songs.length;
+var numberOfSongsOfList1 = collectionPlaylistObjects[1].songs.length;
+var numberOfSongsOfList2 = collectionPlaylistObjects[2].songs.length;
+
+ function showHidePlaylist(id) {
+    var e = document.getElementById(id);
+    e.style.display = (e.style.display == 'block') ? 'none' : 'block';
+ }
+
+
+ function generatePlaylist(idOfPlaylist){
+   var songsOfSpecificPlaylist;
+   var iDiv = document.createElement('div');
+   iDiv.id = 'playlist-container';
+   iDiv.style.cssText = 'position : fixed; background-color : lightblue; height : 680px; width: 470px; margin-left : 50px; margin-top : -620px; overflow-y : scroll;';
+   document.body.appendChild(iDiv);
+
+   switch (idOfPlaylist) {
+    case 0:
+        songsOfSpecificPlaylist = collectionSongObjects.slice(0, numberOfSongsOfList0);
+        break;
+    case 1:
+        songsOfSpecificPlaylist = collectionSongObjects.slice(numberOfSongsOfList0, numberOfSongsOfList0 + numberOfSongsOfList1);
+        break;
+    case 2:
+        songsOfSpecificPlaylist = collectionSongObjects.slice(numberOfSongsOfList0 + numberOfSongsOfList1, numberOfSongsOfList0 + numberOfSongsOfList1 + numberOfSongsOfList2);
+        break;
+  }
+
+
+   for(var i = 0; i < songsOfSpecificPlaylist.length; i++){
```

Comment:
use a  forEach on array instead of "for" . But use it ONLY when map, filter, reduce cannot be used for whatever reason